test/images/image-util.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -x
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TASK=$1
+TEST=$2
+ARCH=${3:-all}
+
+declare -A QEMUARCHS=( ["arm"]="arm" ["arm64"]="aarch64" ["ppc64le"]="ppc64le" ["s390x"]="s390x" )
+echo "${QEMUARCHS[arm64]}"
+
+listArchs() {
+  (
+    cd ${TEST}
+    if [ -f BASEIMAGE ]; then
+      cut -d "=" -f 1 BASEIMAGE
+    fi
+  )
+}
+
+getBaseImage() {
+  arch=$1
+  if [ -f BASEIMAGE ]; then
+    for image in $(cat BASEIMAGE)
+    do
+      file_arch=$(echo ${image} | cut -d "=" -f 1)
+      if [ "${arch}" == "${file_arch}" ]; then
+        echo ${image} | cut -d "=" -f 2
+        break
+      fi
+    done
+  fi
+}
+
+build() {
+  archs=$(listArchs)
+
+  for arch1 in ${archs}; do
+    echo "Build image for ${TEST} ARCH: ${arch1}..."
+    temp_dir=$(mktemp -d)
+    cp -r ${TEST}/* ${temp_dir}
+    if [ -f ${TEST}/bootstrap.sh ]; then
+      pushd ${TEST}
```

Comment:
Why don't just cd into `${temp_dir}` here, run the script and skip the second arg?
Would be much cleaner